index.html

```diff
@@ -84,7 +84,7 @@ <h1 class="show-for-small-only hero-main-header"><img src="{{ site.baseurl }}/im
                     <h1 class="small-12">OME 2021 Community Meeting</h1>
                     <h5 class="small-10 small-offset-1 medium-4 medium-offset-4 bold text-shadow-light">Join us from June 08th to June 11th 2021 for our virtual community meeting.</h5>
                     <br>
-                    <a href="{{ site.baseurl }}/events/ome-community-meeting-2021/" class="large button sites-button btn-red">View the preliminary programme</a>
+                    <a href="{{ site.baseurl }}/events/ome-community-meeting-2021/" class="large button sites-button btn-red">View the final programme</a>
```

Comment:
Ok for this to go live now?

Reply:
Worth confirming with the organizing committee but I would say the program (or at least the times) are as final as can be. The only difference might be more demos/break-out sessions.